command/job_status.go

```diff
@@ -493,6 +507,30 @@ func (c *JobStatusCommand) formatDeployment(client *api.Client, d *api.Deploymen
 	return base
 }
 
+func formatJobActions(actions []string) string {
+	if len(actions) == 0 {
+		return "No actions configured"
+	}
+
+	actionsOut := make([]string, len(actions)+1)
+	actionsOut[0] = "Action Name|Task Group|Task"
+
+	for i, action := range actions {
+		parts := strings.Split(action, "|")
+		if len(parts) != 3 {
+			continue
```

Comment:
hmm, how many of these fields allow "|" in the jobspec?

```hcl
job "jobby" {
  group "hello|cli|output" {
    task "how|are|you" {
      action "pipe|organ" {
...
```

what if instead of `actions []string` it was `actions map[string]string` (or a dedicated little struct), so (see next comment on line 413)